lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -248,17 +253,16 @@ class LLDBExprNameLookup : public LLDBNameLookup {
     if (NameStr.empty())
       return false;
 
-    ConstString name_const_str(NameStr);
-    if (m_log) {
-      m_log->Printf("[LLDBExprNameLookup::lookupAdditions (%u)] Searching for %s",
-                    count, name_const_str.AsCString("<anonymous>"));
-    }
+    if (m_log)
+      m_log->Printf(
+          "[LLDBExprNameLookup::lookupAdditions (%u)] Searching for %s", count,
+          NameStr.empty() ? "<anonymous>" : NameStr.str().c_str());
```

Comment:
`NameStr` can't be empty at this point, because of the early return guard clause just prior.